### DISCUSSION TITLE
I have a lot of errors After upgrading @web3-react/types and @web3-react/core packages.

### DISCUSSION THREAD

**Discussion by deeplearning335** [WalletConnect/walletconnect-monorepo#3651] [Developer Support] [answered]
I have a lot of errors After upgrading @web3-react/types and @web3-react/core packages.
my website is using old react-scripts and typescript.
Thus,after I upgrade the @web3-react/types and @web3-react/core pakcages, I have a lot of errors.
so I can't upgrade the packages.
What should I do?

**Comment by deeplearning335** [comment]
`  const { error } = useWeb3React()`
`const { active } = useWeb3ReactCore()`
`import { UnsupportedChainIdError,} from '@web3-react/core'

**Comment by glitch-txs** [comment, marked as answer]
Hi, this issue is related to web3-react.
If you're using React we recommend using our [Web3Modal SDK](https://docs.walletconnect.com/2.0/web3modal/v3/about) 😄
